src/main/java/com/epam/ta/reportportal/ws/model/settings/GoogleAnalyticsResource.java

```diff
@@ -0,0 +1,52 @@
+package com.epam.ta.reportportal.ws.model.settings;
+
+import com.epam.ta.reportportal.ws.annotations.NotEmpty;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonInclude.Include;
+import com.google.common.base.MoreObjects;
+
+import javax.validation.constraints.NotNull;
+import java.io.Serializable;
+
+@JsonInclude(Include.NON_NULL)
+public class GoogleAnalyticsResource implements Serializable{
```

Comment:
AnalyticsResource. Add field 'type'. For example, of smb decide to switch from google analytics to another one